### DISCUSSION TITLE
Access a room from another room?

### DISCUSSION THREAD

**Discussion by corneliaforslund** [mozilla/hubs#3322] [General Questions & Discussion] [answered]
Hi, If I want to be able to make people be able to go to another room from my first room via the thumbnail, something like the image below, what is the easiest way to do this? 

![Screenshot 2020-11-12 at 10 16 42](https://user-images.githubusercontent.com/74351122/98921333-75ea8280-24d1-11eb-80af-c990543b936b.png)

**Comment by robinkwilson** [comment, marked as answer]
That thumbnail is the easiest way to do it. Just paste a link to the other room in the room + pin it OR via spoke.

Hope this helps.

**Reply by Softvision-GeluHaiduc** [reply]
Marking this as answered, unmark the answer if you need any more information. Thank you!
